tests/tools/onert_train/src/nnfw_util.cc

```diff
@@ -46,4 +49,73 @@ uint64_t bufsize_for(const nnfw_tensorinfo *ti)
   return elmsize[ti->dtype] * num_elems(ti);
 }
 
+std::ostream &operator<<(std::ostream &os, const NNFW_TRAIN_OPTIMIZER &opt)
+{
+  switch (opt)
+  {
+    case NNFW_TRAIN_OPTIMIZER_ADAM:
+      os << "adam";
+      break;
+    case NNFW_TRAIN_OPTIMIZER_SGD:
+      os << "sgd";
+      break;
+    default:
+      os << "unsupported optimizer";
+      break;
+  }
+  return os;
+}
+
+std::ostream &operator<<(std::ostream &os, const NNFW_TRAIN_LOSS &loss)
+{
+  switch (loss)
+  {
+    case NNFW_TRAIN_LOSS_MEAN_SQUARED_ERROR:
+      os << "mean squared error";
+      break;
+    case NNFW_TRAIN_LOSS_CATEGORICAL_CROSSENTROPY:
+      os << "categorical crossentropy";
+      break;
+    default:
+      os << "unsupported loss";
+      break;
+  }
+  return os;
+}
+
+std::ostream &operator<<(std::ostream &os, const NNFW_TRAIN_LOSS_REDUCTION &loss_reduction)
+{
+  switch (loss_reduction)
+  {
+    case NNFW_TRAIN_LOSS_REDUCTION_INVALID:
+      os << "invalid (use default)";
```

Comment:
I've thought about this part. 

Since it is a "dump" function, its role is just printing out the 'nnfw_train_info' value.
Interpreting the 'invalid' meaning as 'use default' or 'use model parameter' is a bit out of this function's role. 

So, Renaming `NNFW_TRAIN_LOSS_REDUCTION_INVALID` into `NNFW_TRAIN_LOSS_REUDCTION_DEFAULT` looks a better option for me.  
After renaming the enum, fixing this dump function to print 'use default option' (instead of printing invalid) looks good to me. 

> So, Renaming `NNFW_TRAIN_LOSS_REDUCTION_INVALID` into `NNFW_TRAIN_LOSS_REUDCTION_DEFAULT` looks a better option for me.   

It is an out of this PR purpose, so let me fix it later.